app/src/main/res/layout/activity_filtering_books.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".FilteringBooksActivity">
+
+<!--    Reset default filtering values-->
+    <Button
+        android:id="@+id/reset_button"
+        android:text= "@string/reset"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="9dp"
+        android:layout_marginEnd="9dp"
+        android:textAllCaps="false"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+<!--    Go to results -->
+    <Button
+        android:id="@+id/results_button"
+        android:text="@string/results"
+        android:layout_width="wrap_content"
```

Comment:
Both Buttons should either be styled as Button, or if not because the size is not right or something like that, extracted into their own new style.

Reply:
Good catch, I forgot about the styled button! Done

Reply:
I haven't checkout the branch directly, but by looking at the xml, it hasn't been done?